go/porcelain/deploy.go

```diff
@@ -579,12 +583,16 @@ func bundle(functionDir string, observer DeployObserver) (*deployFiles, error) {
 				return nil, err
 			}
 			functions.Add(file.Name, file)
-		case goFile(filePath, i):
+		case goFile(filePath, i, observer):
 			file, err := newFunctionFile(filePath, i, goRuntime, observer)
 			if err != nil {
 				return nil, err
 			}
 			functions.Add(file.Name, file)
+		default:
```

Comment:
I'm curious if there is a case that function directory could have some files that are not zip nor js nor go legitimately

Reply:
i think this walks the temporary dir from zisi, therefore only files that zisi picked up should be there